.github/.reviewdogConfig.yml

```
@@ -1,20 +1,20 @@
 runner:
   bandit:
-    cmd:  'bandit -r ./ -f custom --msg-template "{relpath}:{line}:{col}:{test_id}:{severity}:{msg}"' 
+    cmd:  'bandit -r ./ -f custom --msg-template "{relpath}:{line}:{col}:{test_id}:{severity}:{msg}" | tee ./scan_results/banditResults.txt' 
```

suggest using $PWD for absolute path:
```suggestion
    cmd:  'bandit -r ./ -f custom --msg-template "{relpath}:{line}:{col}:{test_id}:{severity}:{msg}" | tee "$PWD"/scan_results/banditResults.txt' 
```